This patch adds support for H27UBG8T2 used on the DC board

diff --git a/drivers/mtd/nand/nand_hynix.c b/drivers/mtd/nand/nand_hynix.c
index 21fb943..c7e528b 100644
--- a/drivers/mtd/nand/nand_hynix.c
+++ b/drivers/mtd/nand/nand_hynix.c
@@ -325,6 +325,10 @@ struct hynix_nand_initializer initializers[] = {
 		.init = h27ucg8t2a_init,
 	},
 	{
+		.id = {NAND_MFR_HYNIX, 0xd7, 0x94, 0xda, 0x74, 0xc3},
+		.init = h27ucg8t2a_init,
+	},
+	{
 		.id = {NAND_MFR_HYNIX, 0xde, 0x14, 0xa7, 0x42, 0x4a},
 		.init = h27q_init,
 	},
